File: modul 2/PRAK205-2410817320004-AmandhaCitraMustika.py
import math
a = float (input())
b = float (input())
c = math.sqrt((b*b) - (a*a))
keliling = a + b + c
luas = 0.5*c*a
print("Alas = %.0f cm" %(c))
print("Tinggi = %.0f cm" %(a))
print("Keliling = %.0f cm" %(keliling))
print("Luas = %.0f cm^2" %(luas))

File: modul 2/PRAK205-2410817320004-AmandhaCitraMustika.c
#include <stdio.h>
#include <math.h>

int main(){
    float a,b,c, keliling, luas;
    scanf("%f", &a);
    scanf("%f", &b);

    c = sqrt ((b*b) - (a*a));
    keliling = a + b + c;
    luas = 0.5*c*a;
    printf("Alas = %.f cm\n", c);
    printf("Tinggi = %.f cm\n", a);
    printf("Keliling = %.f cm\n", keliling);
    printf("Luas = %.f cm^2\n", luas);
    return 0;
}